src/Components/Aspire.Microsoft.EntityFrameworkCore.Cosmos/AspireAzureEFCoreCosmosExtensions.cs

```diff
@@ -48,7 +48,12 @@ public static class AspireAzureEFCoreCosmosExtensions
 
         var settings = builder.GetDbContextSettings<TContext, EntityFrameworkCoreCosmosSettings>(
             DefaultConfigSectionName,
-            (settings, section) => section.Bind(settings)
+            (settings, section) =>
+            {
+                var namedConfigSection = section.GetSection(connectionName);
+                section.Bind(settings);
+                namedConfigSection.Bind(settings);
```

Comment:
I don't think we need to bind by `connectionName` in these EF libraries. That would make 4 places we look:

* Default
* Default:`typeof(TContext).Name`
* Default:`connectionName`
* Default:`typeof(TContext).Name`:`connectionName`


I think we only need to do the first 2.

Reply:
Ah yes, I believe you clarified this for `EnrichDbContext` but I hadn't extended the same philosophy to the `AddDbContext` methods.